packages/scandipwa/src/component/WishlistItem/WishlistItem.component.js

```diff
@@ -237,9 +242,14 @@ export class WishlistItem extends PureComponent {
         const { label, value } = option;
 
         return (
-            <span block="WishlistItemOption" key={ `${ label }-${ value }` }>
-                { `${label }: ${ value}` }
-            </span>
+            <div block="WishlistItem" elem="Option" key={ `${ label }-${ value }` }>
+                <div block="WishlistItem" elem="BundleGroupTitle">
+                    <strong>{ `${label}:` }</strong>
```

Comment:
There is no reason to wrap the text value to the template literals if you don't add any additional values to the string.
Please use this value as is.